editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -479,9 +491,27 @@ export const NavigatorItem: React.FunctionComponent<
   )
   const select = React.useCallback(
     (event: any) => {
-      selectItem(dispatch, getSelectedViewsInRange, navigatorEntry, index, selected, event)
+      if (isConditionalClauseNavigatorEntry(navigatorEntry) && elementMetadata != null) {
```

Comment:
I think we want this logic inside `selectItem`, as it's now split between this handler and the function.

Reply:
Good idea, see 32c6cb04f248c748922f47042198f1bb3bf62d16